packages/amplify-cli/src/__tests__/commands/env.test.ts

```diff
@@ -1,8 +1,17 @@
 describe('amplify env: ', () => {
   const mockExit = jest.fn();
   jest.mock('amplify-cli-core', () => ({
+    ...(jest.requireActual('amplify-cli-core') as Record<string, never>),
     exitOnNextTick: mockExit,
-    pathManager: { getAmplifyMetaFilePath: jest.fn().mockReturnValue('test_file_does_not_exist') },
+    pathManager: {
+      getAmplifyMetaFilePath: jest.fn().mockReturnValue('test_file_does_not_exist'),
+      getAWSCredentialsFilePath: jest.fn(),
+      getAWSConfigFilePath: jest.fn(),
+    },
+    FeatureFlags: {
+      getBoolean: jest.fn().mockReturnValue(true),
+      getNumber: jest.fn().mockReturnValue(1),
+    },
```

Comment:
Why did this need to be added? I don't see any additional uses of FeatureFlags in this PR

Reply:
I'm not entirely sure why the changes necessitated this. Leaving this out resulted in the following.

```
amplify env:  › encountered a declaration exception

    FeatureFlags is not initialized

      211 |   private static ensureInitialized = (): void => {
      212 |     if (!FeatureFlags.instance) {
    > 213 |       throw new Error('FeatureFlags is not initialized');
          |             ^
      214 |     }
      215 |   };
      216 |
```

The requireActual and mocks for path manager were also necessary for separate errors.

I did try mocking amplify-service-helper directly, but it did not prevent the errors.
```
jest.mock('../../amplify-service-helper', () => ({
  ...(jest.requireActual('../../amplify-service-helper') as Record<string, never>),
  postPullCodegen: jest.fn(() => Promise.resolve()),      
}));  
```